scripts/parseFileToReactDoc.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const reactDocs = require("react-docgen");
+
+function typesHandler(documentation) {
+  const componentName = documentation._data.get("displayName").split(".")[0];
+
+  documentation._props.forEach((value, key) => {
+    let newValue = null;
+
+    if (value.type && value.type.name === "enum") {
+      newValue = value.type.value.map(propValue => {
+        if (new RegExp(/^types./).test(propValue.value)) {
+          return { ...propValue, value: `${componentName}.${propValue.value}` };
+        }
+        return { ...propValue };
+      });
+    }
+
+    if (value.defaultValue && new RegExp(/^types./).test(value.defaultValue.value)) {
+      documentation._props.set(key, {
+        ...value,
+        type: {
+          ...value.type,
+          value: newValue,
```

Comment:
I'm not sure when the following could happen but:

wouldn't be better to newValue be set to `undefined` if you set it to null, the defaultValue will be `null` and be pass-down into another component, but if it's undefined the prop will not propagate.